### ISSUE TITLE
No-Member Reported for Abstract Reverse Foreign Keys

### ISSUE THREAD

**Issue by jpulec** [PyCQA/pylint-django#310]
It appears that pylint-django is incorrectly reporting a `no-member` error when trying to reference a reverse related manager from a model that inherits from an abstract model with a foreign key.

For example:

```python

class Owner(models.Model):
    pass


class OwnerMixin(models.Model):
    owner = models.ForeignKey(Owner, on_delete=models.CASCADE, related_name="%(app_label)s_%(class)s")
    
    class Meta:
         abstract = True


class BookManager(models.Manager):
    def special_books(self, *args, **kwargs):
        pass

class Book(OwnerMixin):
    objects = BookManager()



owner = Owner.objects.first()

owner.books.special_books()  # <-- no-member error throw here
```
